test/connection.test.ts

```diff
@@ -520,6 +527,30 @@ describe('Connection', function () {
       assert.calledOnce(spy)
       assert.calledWith(spy, '58')
     })
+
+    it('should get a receipt for each fulfilled packet', async function () {
+      let streamStartTime: Long
+      this.serverConn.on('stream', async (serverStream: DataAndMoneyStream) => {
+        streamStartTime = serverStream.startTime
+      })
+      const clientStream = this.clientConn.createStream()
+      const spy = sinon.spy(clientStream, '_setReceipt')
+      await clientStream.sendTotal(1002)
+
+      async function createReceipt(receiptNonce: Buffer, receiptSecret: Buffer, totalReceived: string): Promise<Buffer> {
+        const receipt = new Writer(65)
+        receipt.writeOctetString(receiptNonce, 16)
+        receipt.writeUInt8(clientStream.id)
+        receipt.writeUInt64(longFromValue(totalReceived, true))
+        receipt.writeUInt64(longFromValue(streamStartTime, true))
+        receipt.writeOctetString(await hmac(receiptSecret, receipt.getBuffer()), 32)
+        return Promise.resolve(receipt.getBuffer())
+      }
+
+      assert.calledTwice(spy)
+      assert.calledWith(spy.firstCall, await createReceipt(this.receiptNonce, this.receiptSecret, '500'))
```

Comment:
Maybe add an assertion that compares the receipt to a fixed (hard-coded) value.

Reply:
updated it to use the fixture: https://github.com/interledgerjs/ilp-protocol-stream/pull/133/commits/96d786dd5b62ab81ea3280876d4a6bcfdb4bfc9c